app/lib/submission_builder/ty2024/states/md/documents/md502.rb

```diff
@@ -35,11 +47,23 @@ def calculated_fields
               @md502_fields ||= intake.tax_calculator.calculate
             end
 
+            def has_dependent_exemption
+              [
+                :MD502_DEPENDENT_EXEMPTION_COUNT,
+                :MD502_DEPENDENT_EXEMPTION_AMOUNT
+              ].any? do |line|
+                calculated_fields.fetch(line) > 0
+              end
+            end
+
+            def has_exemptions
+              has_dependent_exemption
```

Comment:
is this a separate method right now because there might be other exemptions?

Reply:
that was the thought! i did intend to ask though so this is a good reminder haha

Reply:
https://codeforamerica.atlassian.net/browse/FYST-738?focusedCommentId=12290